drivers/rptun/Kconfig

```diff
@@ -12,6 +12,12 @@ menuconfig RPTUN
 
 if RPTUN
 
+config RPTUN_SECURE
+	bool "rptun secure support"
+	default n
+	---help---
+		This is for secure rptun support.
```

Comment:
Please include more information here. Why is it considered secure? What is main differences?

Reply:
Done